src/test/resources/rca/bucketization/bucketize_no_thresholds.json

```diff
@@ -0,0 +1,4 @@
+{
+  "bucketization": {
+  }
+}
```

Comment:
Can we use a string (in the same file) and pass it to rca.conf 

Ref:
https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyCacheMaxSizeActionTest.java#L113